src/deepsparse/transformers/utils/eval_helpers.py

```diff
@@ -0,0 +1,94 @@
+from transformers import AutoTokenizer
+from datasets import load_dataset
+import numpy
+
+
+def process_concatenated_datasets(dataset_name, model_path, max_sequence_length, kwargs):
```

Comment:
docstring, typing

Reply:
@anmarques was this adapted from somewhere or new code we are writing? If adapted, let's get a link in.  In either case let's make sure to include a docstring that specifies what is returned and the scope of the function

Reply:
Fixed